modules/zio2/logging/src/main/scala/tofu/logging/impl/ZContextLogging.scala

```diff
@@ -0,0 +1,34 @@
+package tofu.logging.impl
+
+import org.slf4j.{Logger, Marker}
+import tofu.logging.zlogs.ZLogging
+import tofu.logging.{Loggable, LoggedValue, Logging}
+import zio._
+
+class ZContextLogging[R, Ctx: Loggable](logger: => Logger, ctxLog: URIO[R, Ctx]) extends ZLogging[R] {
```

Comment:
>logger: => Logger

why by-name?

Reply:
Discussed with @Odomontois this `by-name` is OK